ui/src/app/components/inputs.tsx

```diff
@@ -24,14 +37,37 @@ export const styles = {
     lineHeight: '1.5rem',
     height: '1.5rem',
     width: '16rem'
+  },
+
+  errorMessage: {
+    width: '12.5rem',
+    padding: '.25rem',
+    float: 'right' as 'right',
+    marginTop: 0,
+    background: '#f5dbd9',
+    color: '#565656',
+    border: '1px solid #ebafa6',
+    display: 'flex' as 'flex',
+    flexDirection: 'row' as 'row',
+    fontSize: '13px'
+  },
+
+  iconArea: {
+    display: 'inline-block',
+    marginLeft: '-30px',
+    minWidth: '30px'
   }
 };
 
-
 export const FieldInput = ({style = {}, ...props}) =>
   <input {...props} style={{...styles.input, ...style}} />;
 export const FormInput = ({style = {}, ...props}) =>
     <input {...props} style={{...styles.formInput, ...style}} ref={props.inputref}/>;
+export const LongInput = ({style = {}, ...props}) =>
+  <input {...props} style={{...styles.formInput, ...styles.longInput, ...style}} />;
 export const Error = ({style = {}, ...props}) =>
   <div {...props} style={{...styles.error, ...style}} />;
-
+export const FormSection = ({style = {}, ...props}) =>
+  <div {...props} style={{...styles.formSection, ...style}} />;
```

Comment:
So from the class name i assumed the styles will be for inputs and not form. I think we should change the name of the file or movie Formsection error message to a different file

Reply:
Yeah I almost did, but currently this is the only "form" element, so felt silly to do another whole file, plus this would only be used in conjunction with inputs... If anyone really has strong feeling about this I'll move it.